assets/templates/plan-calculator.html

```diff
@@ -101,14 +101,213 @@ <h1 class="dp-tit-plans">Plan STANDARD</h1>
+                                  type="button"
+                                  class="dp-button button-medium"
+                                >
+                                  Mensual
+                                </button>
+                              </li>
+                              <li>
+                                <button
+                                  type="button"
+                                  class="dp-button button-medium"
+                                >
+                                  Trimestral
+                                </button>
+                                <span class="dp-discount">5% off</span>
+                              </li>
+                              <li>
+                                <button
+                                  type="button"
+                                  class="dp-button button-medium"
+                                >
+                                  Semestral
+                                </button>
+                                <span class="dp-discount">15% off</span>
+                              </li>
+                              <li>
+                                <button
+                                  type="button"
+                                  class="dp-button button-medium btn-active"
+                                >
+                                  Anual
+                                </button>
+                                <span class="dp-discount">25% off</span>
+                              </li>
+                            </ul>
+                          </div>
+
+                          <div class="dp-calc-message dp-show">
+                            <p>
+                              ¿Tienes más de 100.000 contactos?
+                              <a href="#"
+                                ><strong
+                                  >Contáctanos para crear un plan a tu
+                                  medida.</strong
+                                ></a
+                              >
+                            </p>
+                          </div>
+                        </article>
+                      </div>
+                      <div class="col-md-6 col-sm-12">
+                        <div class="dp-wrap-price">
+                          <span class="dp-old-price"
+                            ><span class="dp-money">US$</span
+                            ><span class="dp-old-amount">15</span></span
+                          >
+                          <h2 class="dp-amount-large">
+                            <span class="dp-money">US$</span
+                            ><span class="dp-amount">11,25</span>
+                          </h2>
+                          <!--<h2 class="dp-amount-small">
+                          <span class="dp-money">Bs</span
+                          ><span class="dp-amount">85.728.236,58</span>
+                        </h2>-->
```

Comment:
I thinks this whole section is referred to price, perhaps the naming of classes could be something like this:
 
```suggestion
                        <div class="dp-price--wrapper">
                          <span class="dp-price-old"
                            ><span class="dp-price-old--money">US$</span
                            ><span class="dp-price-old--amount">15</span></span
                          >
                          <h2 class="dp-price-large">
                            <span class="dp-price-large--money">US$</span
                            ><span class="dp-price-large--amount">11,25</span>
                          </h2>
                          <!--<h2 class="dp-price-small">
                          <span class="dp-price-small--money">Bs</span
                          ><span class="dp-price-small--amount">85.728.236,58</span>
                        </h2>-->
```
In this way I believe this has more meaning as component, one whole component dp-price with variations for size and modificators (after --) money and amount.

Reply:
Excellent!

Reply:
ready the change! 👍